sri/utils.py

```diff
@@ -14,10 +17,12 @@ def get_static_path(path: str) -> Path:
 
     if hasattr(staticfiles_storage, "stored_name"):
         path = staticfiles_storage.stored_name(path)
-        collected_file_path = staticfiles_storage.path(path)
-        if os.path.exists(collected_file_path):
-            return Path(collected_file_path)
 
+    collected_file_path = staticfiles_storage.path(path)
+    if os.path.exists(collected_file_path):
+        return Path(collected_file_path)
```

Comment:
This conditional is only true for manifest storage; if you are using other storage classes then this function will currently return the source file, not the collected static file.